### DISCUSSION TITLE
Scroll Lock Key?

### DISCUSSION THREAD

**Discussion by lautarodragan** [kovidgoyal/kitty#4749] [Ideas]
This key is pretty much dead, but considering it still comes in keyboards, it'd be nice to use it. I _think_ its intended functionality is to cause the arrow keys to scroll. It'd basically trigger the functionality that's already implemented, currently triggered by the mouse wheel.

Is this implemented? Can this be configured? Or is this a feature request? If the latter, is anyone else interested?

------

I seem to have a rather old version... I hope I'm not missing out

```
$ kitty -v
kitty 0.19.3 created by Kovid Goyal
```

**Comment by kovidgoyal** [comment]
IIRC scrolllock works fine in kitty. map it to do whatever you want in kitty.conf. Or it will work out of th ebox with applications that support kitty's advanced keyboard protocol. And yes your kitty version is ancient, update: https://sw.kovidgoyal.net/kitty/binary/

**Reply by lautarodragan** [reply]
Oki, thanks! Kitty's EXCELLENT by the way, and I absolutely love it. Thank you for it.
